abi/src/test/java/org/web3j/abi/TypeDecoderTest.java

```diff
@@ -177,11 +215,19 @@ public void testStaticBytes() {
         assertThat(TypeDecoder.decodeBytes(
                 "6461766500000000000000000000000000000000000000000000000000000000", Bytes4.class),
                 is(dave));
+        try {
```

Comment:
Please remove all try/catch blocks from tests (see comment below) and also `printStackTrace` calls.

Reply:
done

Reply:
The `try`...`catch` comment relates to all tests and there are many of them still there.

Reply:
Eh, looks like all try/catch were removed from https://github.com/streamr-dev/web3j/blob/master/abi/src/test/java/org/web3j/abi/TypeDecoderTest.java. The code snippet above is outdated. This PR only modified TypeDecoderTest.

What are you referring to?

Reply:
Sorry my bad, if there are no try/catch blocks in the test that's fine for me.